public/js/action-list.js

```diff
@@ -82,14 +82,25 @@
                 $list.find('.list-item.active').removeClass('active');
                 $item.addClass('active');
             }
+
+            // For items that do not have a bottom status bar like Downtimes, Comments...
+            if ($('#col1.container').find('.footer > .status-bar').length === 0) {
+                $('#col1').append('<div id="automatically-added" class="footer"><div class="status-bar"></div></div>');
```

Comment:
* Please use `$list.closest('.container')` instead of `$('#col1.container')` or `$('#col1')`
  * It's not guaranteed that the list the user is interacting with resides in `#col1`
* Please don't check for/add the `.status-bar` container, just check if there's a footer
  * Add the selection count to its own container instead
* Don't add an id to the footer
  * An id is only required based on the content. No content, no id